attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/model/AttributeMetadataModel.java

```diff
@@ -53,6 +53,9 @@ public class AttributeMetadataModel implements Document {
   @JsonProperty(value = "display_name")
   private String displayName;
 
+  @JsonProperty(value = "description")
+  private String description;
```

Comment:
The existing documents will not have this property. We need to check if it defaults to null in that case.

Reply:
This should be fine as `ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)`, but lets see if we have a test for it.